src/components/Viewport.ts

```diff
@@ -700,16 +729,78 @@ export default class Viewport {
     this.moveToDefaultPanel();
   }
 
+  private setElements(): void {
+    const state = this.state;
+    const options = this.options;
+    const wrapper = this.flicking.getElement();
+    const classPrefix = options.classPrefix;
+
+    const viewportCandidate = wrapper.children[0] as HTMLElement;
+    const hasViewportElement = hasClass(viewportCandidate, `${classPrefix}-viewport`);
+
+    const viewportElement = hasViewportElement
+      ? viewportCandidate
+      : document.createElement("div");
+
+    const cameraCandidate = hasViewportElement
+      ? viewportElement.children[0] as HTMLElement
+      : wrapper.children[0] as HTMLElement;
```

Comment:
@WoodNeck 
Is there a camera under the wrapper?

Reply:
I've considered all 4 possibilities.
Which is,
1) No Viewport, No Camera element
2) Viewport element exists, while no camera element.
3) Camera element exists, while no viewport element.
4) Both viewport and camera element exist.